securebanking-openbanking-uk-rs-backoffice-api/src/main/java/com/forgerock/securebanking/openbanking/uk/rs/api/backoffice/payment/calculate/elements/v3_1_5/domesticpayments/CalculateResponseElements.java

```diff
@@ -44,17 +45,14 @@ public interface CalculateResponseElements {
     @RequestMapping(value = "/calculate-elements",
             produces = {"application/json; charset=utf-8"},
             method = RequestMethod.POST)
-    ResponseEntity calculateElements(
+    OBWriteDomesticConsentResponse5 calculateElements(
```

Comment:
Needs to be ResponseEntity<T> `Extension of HttpEntity that adds a HttpStatus status code. Used in RestTemplate as well @Controller methods.`
- ResponseEntity<OBWriteDomesticConsentResponse5>

Reply:
Changes implemented